src/overlays/actors/ovl_Bg_Zg/z_bg_zg.h

```diff
@@ -3,10 +3,12 @@
 
 #include <ultra64.h>
 #include <global.h>
+#include <vt.h>
 
 typedef struct {
-    /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x20];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x0164 */ s32 funcIndex;
+    /* 0x0168 */ s32 unk_168;
```

Comment:
0x164 would be `action` and 0x168 would be `drawConfig`, with local variables updated in the different functions where they are used.